lang/tags/functions.py

```diff
@@ -11,6 +13,12 @@
 mod.list(
     "code_parameter_name", desc="List of common parameter names for active language"
 )
+mod.list(
+    "code_function_modifier",
+    desc="List of function modifiers (e.g. private, async, static)",
+)
+
+ctx.lists["user.code_function_modifier"] = ["pub", "pro", "private", "static"]
```

Comment:
ok sorry i'm nitpicking a bit here, but I think I'd prefer to use more verbose internal identifiers
```suggestion
ctx.lists["user.code_function_modifier"] = {"pub": "public", "pro": "protected", "private": "private", "static": "static",}
```